doc/pages/regex.asciidoc

```diff
@@ -62,15 +62,15 @@ Supported character class escapes are:
 * `\h` which matches all horizontal whitespace characters.
 
 Using an upper case letter instead of a lower case one will negate
-the character class, meaning for example that `\D` will match every
-non-digit character.
+the character class. For example, `\D` will match every non-digit
+character.
 
 Character class escapes can be used outside of a character class, `\d`
 is equivalent to `[\d]`.
 
 == Any character
 
-`.` matches any character, including new lines.
+`.` matches any character, including newlines, by default.
```

Comment:
Might want to drop a reference to `:doc regex modifiers` to help people figure out how to change the default.